build.gradle

```diff
@@ -244,6 +236,32 @@ task('format') {
     dependsOn subprojects.collect { it.getTasksByName('airbytePythonFormat', true) }
 }
 
+// add licenses for python projects.
+subprojects {
+    def pythonFormatTask = project.tasks.findByName('blackFormat')
+
+    if(pythonFormatTask != null) {
+        apply plugin: "com.github.hierynomus.license"
+        task licenseFormatPython(type: com.hierynomus.gradle.license.tasks.LicenseFormat) {
+            header = createPythonLicenseWith(rootProject.file('LICENSE'))
+            source = fileTree(dir: projectDir)
+                    .include("**/*.py")
+                    .exclude(".venv/**/*.py")
+                    .exclude("**/__init__.py")
```

Comment:
maintaining existing behavior where we do license __init__ files. because they are blank it seems to cause havoc on these license tools.